tests/test_stub.nf.test.snap

```diff
@@ -2,105 +2,7 @@
-                },
-                "FUSIONCATCHER": {
-                    "fusioncatcher": 1.35
-                },
-                "FUSIONCATCHER_BUILD": {
-                    "fusioncatcher": "fusioncatcher.py 1.35"
-                },
-                "FUSIONREPORT": {
-                    "fusion_report": "2.1.5"
-                },
-                "FUSIONREPORT_DOWNLOAD": {
-                    "fusion_report": "2.1.5"
-                },
-                "GATK4_BEDTOINTERVALLIST": {
-                    "gatk4": "4.6.1.0"
-                },
-                "GATK4_CREATESEQUENCEDICTIONARY": {
-                    "gatk4": "4.6.1.0"
-                },
-                "GATK4_MARKDUPLICATES": {
-                    "gatk4": "4.5.0.0",
-                    "samtools": "1.19.2"
-                },
-                "GENCODE_DOWNLOAD": {
-                    "wget": null
-                },
-                "GET_RRNA_TRANSCRIPTS": {
-                    "get_rrna_transcripts": "v1.0"
-                },
-                "GFFREAD": {
-                    "gffread": "0.12.7"
-                },
-                "GTF_TO_REFFLAT": {
-                    "gtfToGenePred": 377
-                },
-                "HGNC_DOWNLOAD": {
-                    "wget": null
-                },
-                "PICARD_COLLECTINSERTSIZEMETRICS": {
-                    "picard": "3.3.0"
-                },
-                "PICARD_COLLECTRNASEQMETRICS": {
-                    "picard": "3.3.0"
-                },
-                "SALMON_INDEX": {
-                    "salmon": "1.10.3"
-                },
-                "SALMON_QUANT": {
-                    "salmon": "1.10.3"
-                },
-                "SAMTOOLS_FAIDX": {
-                    "samtools": 1.21
-                },
-                "SAMTOOLS_INDEX_FOR_STARFUSION": {
-                    "samtools": 1.21
-                },
-                "STARFUSION": {
-                    "STAR-Fusion": "1.14.0"
-                },
-                "STARFUSION_BUILD": {
-                    "STAR-Fusion": "1.14.0"
-                },
-                "STAR_FOR_ARRIBA": {
-                    "star": "2.7.11b",
-                    "samtools": 1.2,
-                    "gawk": "5.1.0"
-                },
-                "STAR_FOR_STARFUSION": {
-                    "star": "2.7.11b",
-                    "samtools": 1.2,
-                    "gawk": "5.1.0"
-                },
-                "STAR_GENOMEGENERATE": {
-                    "star": "2.7.11b",
-                    "samtools": 1.2,
-                    "gawk": "5.1.0"
-                },
-                "STRINGTIE_MERGE": {
-                    "stringtie": "2.2.1"
-                },
-                "STRINGTIE_STRINGTIE": {
-                    "stringtie": "2.2.3"
-                },
-                "Workflow": {
-                    "nf-core/rnafusion": "v4.0.0dev"
-                }
-            },
+            null,
```

Comment:
@atrigila, @nschcolnicov, @alanmmobbs93: could you check if that is a problem?

Reply:
In the tests, we obtain the versions from:
```
    // pipeline versions.yml file for multiqc from which Nextflow version is removed because we tests pipelines on multiple Nextflow versions
                    removeNextflowVersion("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml"),
```
which has now changed to:

```
name: 'nf_core_' + 'rnafusion_software_' + 'mqc_' + 'versions.yml',
```

in: `workflows/rnafusion.nf`.

So the solution would be to modify this line in the test:
`removeNextflowVersion("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml")` for the correct new version name (`nf_core_rnafusion_software_mqc_versions.yml`).

Reply:
Perfect thanks!